lms/djangoapps/verify_student/tests/__init__.py

```diff
@@ -62,7 +62,7 @@ def verification_active_at_datetime(self, attempt):
 
     def submit_attempt(self, attempt):
         with self.immediate_on_commit():
-            attempt.submit()
+            attempt.mark_submit()
```

Comment:
This might be the reason for your test failures, this should probably be left alone